test/test-applications/integrations/TestApplication.ContinuousProfiler/Exporter/SampleNativeFormatParser.cs

```diff
@@ -8,19 +8,24 @@ namespace TestApplication.ContinuousProfiler;
 /// <summary>
 /// Parser the native code's pause-time-optimized format.
 /// </summary>
-internal static class SampleNativeFormatParser
+public class SampleNativeFormatParser
 {
     // TODO use value from ContinuousProfilerProcessor.BackgroundThreadName when it will be moved to main project
     public const string BackgroundThreadName = "OpenTelemetry Continuous Profiler Thread";
-
     private static readonly UnicodeEncoding UnicodeEncoding = new();
+    private readonly bool _frequentSamplingEnabled;
+
+    public SampleNativeFormatParser(bool frequentSamplingEnabled = false)
```

Comment:
profiling is still unstable, probably only used by Splunk, feel free to include breaking change without any conditions.